components/builder-worker/src/runner/mod.rs

```diff
@@ -314,10 +304,14 @@ impl Runner {
         self.log_pipe().pipe(&mut child)?;
         let exit_status = child.wait().expect("failed to wait on child");
         debug!("build complete, status={:?}", exit_status);
+        fs::rename(self.workspace.src().join("results"), self.workspace.out())?;
```

Comment:
I don't _think_ this will be a problem, because my code that adds the `pre_build.env` file creates that "results" dir if it doesn't already exist, and it does that very close to the beginning of where the hab-plan-build script actually starts doing things.  That said, your suggestion is still a good idea, so I'll make that change.